src/hooks/useLegacyUIURL.ts

```diff
@@ -24,10 +24,7 @@ export const useLegacyUIURL = (): string | null => {
     };
     const legacyUIKeys = Object.keys(legacyUIMap);
     for (let i = 0; i < legacyUIKeys.length; i++) {
-      const matchedPath = matchPath(pathname, {
-        path: legacyUIKeys[i],
-        exact: true,
-      });
+      const matchedPath = matchPath(legacyUIKeys[i], pathname);
```

Comment:
Switch to legacy ui does not link the correct project.
This pr
<img width="1435" alt="image" src="https://user-images.githubusercontent.com/4605522/171485654-48f17bf8-5b20-4ef4-b4d9-062f16dd5419.png">
Link is on Bottom left

Prod
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/4605522/171485724-90506b7b-14f9-4413-862c-27aa192b5c28.png">

Link is on Bottom Left